### ISSUE TITLE
{Linked:Bug11089156} [Adaptive Cards > Designer] - Narrator is not announcing the label names of the 'Need to add an Interim Approver?' and 'Update CC/IO' edit fields.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8703]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/explorer/Action.ToggleVisibility.html
OS Version: 23H3(OS Build 25951.1010)
Browser Version: Edge Dev 117.0.2033.0
Screen Reader: Narrator

**Pre-Requisite:**
Run the below attached JSON file under 'Copy Card Payload editor.'

**Repro Steps:**
1. Open the above URL on the EDGE Dev browser.
2. Run the Json file.
3. Navigate to the Preview mode and press enter key to activate it.
4. Navigate the screen and observe the issue.

**Actual Result:**
Narrator is not announcing the label names of the 'Need to add an Interim Approver?' and 'Update CC/IO' edit fields.
Narrator is announcing as 'Enter Aliases separated by semicolon (;) edit and enter a CC/IO edit 10257098 required 10257098'.

**Expected Result:**
Narrator should announce the label names of the 'Need to add an Interim Approver?' and 'Update CC/IO' edit fields.
Narrator should announce as 'Need to add an Interim Approver? Enter Aliases separated by semicolon (;) edit and Update CC/IO edit 10257098'.

**User Impact:**
Visually challenged users who rely on screen readers are getting impacted, as Narrator is not announcing the label names of the 'Need to add an Interim Approver?' and 'Update CC/IO' edit fields.
 
**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/labels-or-instructions.html

Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-chisha%40microsoft.com%7c1334b110f98744a7f5df08d968eb842b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656183104980596%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=r4ndSHk/LrBDwzi7FO%2BjKN0DCnqUSUcELBvGRkgc5CU%3D&reserved=0)"

### Screenshots

https://github.com/microsoft/AdaptiveCards/assets/95913748/2d389ab3-3e76-4548-a29c-2d20bf7a6f4d

[Test JSON.docx](https://github.com/microsoft/AdaptiveCards/files/12847995/Test.JSON.docx)

![WCAG3 3 2- Screen Reader is not announcing the label name](https://github.com/microsoft/AdaptiveCards/assets/95913748/bc44427a-d9ef-4eb9-8130-1061f89b8caf)

### Card JSON

```JSON
Attached in Screenshot Section
```

**Comment by vagpt** [comment]
Originally tracked using below URL.

https://dev.azure.com/microsoftit/OneITVSO/_workitems/edit/11089156

**Comment by anna-dingler** [comment]
Hi @vagpt,

To resolve this issue, the card author needs to use `Input.Text` `label` property instead of a `TextBlock`. With the label property, Narrator will properly announce the input field.

Let me know if this doesn't work for you!
Anna

**Comment by vagpt** [comment]
Adding A11yTTValidated tag as per the Anna comment,
